bindings/ios/MEGASdk.h

```diff
@@ -5019,6 +5043,81 @@ typedef NS_ENUM(NSUInteger, StorageState) {
  */
 - (void)setUploadLimitWithBpsLimit:(NSInteger)bpsLimit;
 
+/**
+ * @brief Initial step to upload a photo/video via iOS low-power background upload feature
+ *
+ * Call ensureMediaInfo() first in order prepare the library to attach file attributes
+ * that enable videos to be identified and played in the web browser.
+ *
+ * @return A pointer to an object that keeps some needed state through the process of
+ *         uploading a media file via iOS low power background uploads (or similar).
+ *         Caller takes ownership of the object.
+ */
+- (MEGABackgroundMediaUpload *)backgroundMediaUpload;
+
+/**
+ * @brief Get back the needed MEGABackgroundMediaUpload after the iOS app exited and restarted
+ *
+ * In case the iOS app exits while a background upload is going on, and the app is started again
+ * to complete the operation.  Call this version to recreate the MegaBackgroundMediaUpload object
+ * needed for a call to backgroundMediaUploadComplete.   The object must have been serialised
+ * before the app was unloaded.
+ *
+ * @return A pointer to an object that keeps some needed state through the process of
+ *         uploading a media file via iOS low power background uploads (or similar).
+ *         Caller takes ownership of the object.
+ */
+- (MEGABackgroundMediaUpload *)resumeBackgroundMediaUploadBySerializedData:(NSData *)data;
+
+/**
+ * @brief Request the URL suitable for uploading a media file.
+ *
+ * This function requests the URL needed for uploading the file.  The URL will need the urlSuffix
+ * from the MEGABackgroundMediaUpload::encryptFile to be appended before actually sending.
```

Comment:
Usually we use the name of the Obj-c binding instead the name of the c++ intermediate layer, for methods, enums, etc
[MEGABackgroundMediaUpload encryptFileAtPath:startPosition:length:outputFilePath:urlSuffix:adjustsSizeOnly:]

Reply:
fixed